src/KeyboardTrait.php

```
@@ -159,15 +160,56 @@ public function keyboardPressKeyOnElementSingle(string $char, ?string $selector)
+              if (sibling.nodeType === 1 && sibling.tagName === element.tagName)
+                ix++;
+            }
+          }
+
+          return getPathTo(el);
+        })()
+JS;
+      $xpath = $this->getSession()->evaluateScript($script);
+
+      if (!$xpath) {
+        throw new \RuntimeException('No element is currently focused. Please focus an element first using a step with a selector.');
```

PHPMD (2.15.0)

Missing class import via use statement (line '197', column '19'). (undefined)

(MissingImport)